app/lib/custom_log_formatter.rb

```diff
@@ -57,14 +57,3 @@ def method_is_post_or_put_or_patch?
     hash.dig(:payload, :method).in?(%w[PUT POST PATCH])
   end
 end
-
-unless Rails.env.local?
-  Rails.application.configure do
-    config.semantic_logger.application = Settings.application_name
```

Comment:
Removed this line because `Settings.application_name` is nil in all environments.